cmd/integrationArtifactDeploy.go

```diff
@@ -86,13 +88,18 @@ func runDeployIntegrationArtifact(config *deployIntegrationArtifactOptions, tele
 		return errors.Errorf("did not retrieve a HTTP response")
 	}
 
-	if deployResp.StatusCode == 202 {
+	if deployResp.StatusCode == http.StatusAccepted {
 		log.Entry().
 			WithField("IntegrationFlowID", config.IntegrationFlowID).
 			Info("successfully deployed into CPI runtime")
 		return nil
 	}
+	response, readErr := ioutil.ReadAll(deployResp.Body)
```

Comment:
`ioutil.ReadAll()` is only introduced to improve the error handling?

Reply:
yes as per marcus inputs